applications/visualizer/frontend/src/components/viewers/ThreeD/ThreeDViewer.tsx

```diff
@@ -36,6 +38,7 @@ function ThreeDViewer() {
     const [showSynapses, setShowSynapses] = useState<boolean>(true);
     const [instances, setInstances] = useState<Instance[]>([])
     const [isWireframe, setIsWireframe] = useState<boolean>(false)
+    const { datasets} = useGlobalContext();
```

Comment:
question (non-blocking): Shouldn't this be the active datasets (the ones that are active in the current workspace), opposed to all the existent datasets?